src/processTargets/processMark.ts

```diff
@@ -134,10 +133,7 @@ function processDecoratedSymbol(
   context: ProcessedTargetsContext,
   mark: DecoratedSymbol
 ) {
-  const token = context.hatTokenMap.getToken(
-    mark.symbolColor,
-    mark.character
-  );
+  const token = context.hatTokenMap.getToken(mark.symbolColor, mark.character);
```

Comment:
Btw is this stuff enforced as of this PR?  If not, we really have to get these linting checks into CI 😅

Reply:
I haven't touched anything relating to formatting

Reply:
Ok.  Yeah we gotta get these checks into CI so we don't keep having all these polluted diffs.  I'll see if we have an issue for it

Reply:
Sounds great